docs/source/cli_help/update.rst

```diff
@@ -0,0 +1,27 @@
+update
+---------
+Usage:
+::
+    git machete update [-f\|--fork-point=] [-M\|--merge] [-n\|--no-edit-merge\|--no-interactive-rebase]
```

Comment:
Hmm looks that `|`s don't need to be escaped:

![image](https://user-images.githubusercontent.com/3383210/131721868-a7ebf2bf-efc5-4542-b13a-ea00eef487eb.png)